api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -400,7 +400,7 @@ private DbWorkspace createDbWorkspace(
       workspaceService.updateWorkspaceBillingAccount(
           dbWorkspace, workspace.getBillingAccountName());
     } catch (ServerErrorException e) {
-      new ServerErrorException("Could not update the workspace's billing account", e);
```

Comment:
unrelated to recent changes, but swallowing these exceptions looked like obvious bugs worth fixing to me